lib/components/header_bar.dart

```diff
@@ -11,8 +11,7 @@ class _DurationWidget extends StatelessWidget {
   final DateTime from;
   final TextStyle? style;
 
-  const _DurationWidget({Key? key, this.style, required this.from})
-      : super(key: key);
+  const _DurationWidget({this.style, required this.from});
```

Comment:
Missing `super.key`.

Reply:
A value for optional parameter 'key' isn't ever given.
Try removing the unused parameter.